testsuite/CMakeLists.txt

```diff
@@ -5,4 +5,8 @@ add_subdirectory(tabulated)
 add_subdirectory(system_test)
 add_subdirectory(minimize_energy)
 add_subdirectory(Array4DTest)
-add_subdirectory(fixed_list_types_test)
\ No newline at end of file
+add_subdirectory(fixed_list_types_test)
+add_subdirectory(RDFatomistic)
+add_subdirectory(AdressDensity)
+add_subdirectory(TDforce)
+add_subdirectory(FreeEnergyCompensation)
```

Comment:
Possibly a good idea. How's that being done properly? Do I need to add just another CMakeLists.txt like this one in this subfolder specifying the different folders like here?